src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -423,6 +423,13 @@ public override TypeInfo GetTypeInfo(SelectOrGroupClauseSyntax node, Cancellatio
             return (model == null) ? default(TypeInfo) : model.GetTypeInfo(node, cancellationToken);
         }
 
+        public override SymbolInfo GetSymbolInfo(SubPropertyPatternSyntax node, CancellationToken cancellationToken)
+        {
+            CheckSyntaxNode(node);
+            var model = this.GetMemberModel(node);
+            return (model == null) ? default(SymbolInfo) : model.GetSymbolInfo(node, cancellationToken);
```

Comment:
That would be different from all the other places this pattern appears in the code. I'd rather be consistent and review a possible change all at once.